openpype/hosts/max/plugins/publish/extract_max_scene_raw.py

```diff
@@ -33,6 +33,7 @@ def process(self, instance):
             instance.data["representations"] = []
 
         nodes = instance.data["members"]
+        nodes.append(container)
```

Comment:
This will affect content of `instance.data["members"]` permanently (any future access to that key will contain the added node too).

NOTE: I'm asking if it's ok, I don't know if it's expected behavior.

Reply:
you actually raise a good point. it affect content of the instance.data["members"] if there is multiple extractor working.